packages/blueprints-integration/src/rundown.ts

```diff
@@ -166,6 +166,15 @@ export interface IBlueprintSegment<TMetadata = unknown> {
 	showShelf?: boolean
 	/** Segment display mode. Default mode is *SegmentDisplayMode.Timeline* */
 	displayAs?: SegmentDisplayMode
+
+	/** Used to alter the behavior of the Diff timer to make it relative to this segment's hard backtime. If true, `breakStartTime` and/or `breakEndTime` must also be set. */
+	isBreak?: boolean
+
+	/** Does nothing without `isBreak`. */
+	breakStartTime?: number
+
+	/** Does nothing without `isBreak`. */
+	breakEndTime?: number
```

Comment:
I think this functionality can be used for so much more than breaks, so I'd rather these properties be called `expectedStart` and `expectedEnd` as is convention in all other timing-related properties in Sofie, since this is what they describe: the "expected time something _should_ happen". It then makes the clear distinction between plan and execution.

Reply:
Also, I don't believe the comment about `isBreak` needing to be `true` is correct, based on the implementation, and rightfully so. If the user wants to anchor this Segment in time, they can use either of these properties to signify so. I don't see the need for another property. The comments for the two properties could be also made a bit more clear, like: `Expected end time of this Segment`.